pages/api/goals/goalsHelpers.ts

```diff
@@ -0,0 +1,12 @@
+import { ReactElement } from "react";
+import { UserGoalsData } from "../../../graphql/fetchUserGoals";
+
+export type GoalsSection = {
+  header: GoalsSectionHeader;
+  rows: UserGoalsData[];
+};
+
+export type GoalsSectionHeader = {
+  sectionName: string;
+  manageIcons: Array<ReactElement>;
```

Comment:
is manageIcons really part of the Header? seems like it represents the icons you want to display for each row in this section. 

I might remove this type entirely, and move both `sectionName` and `manageIcons` into the `GoalsSection` type

Reply:
not a big deal either way, im fine leaving it as is.